src/components/common/NameRequestInfo.vue

```diff
@@ -155,6 +154,51 @@
         </v-row>
       </div>
     </template>
+    <template v-else>
```

Comment:
Is this a continuation of the v-else-if on line 100? If so, then THIS block will be rendered if "this is not an amalgamation filing" or "there is a name request number". This logic seems incorrect; should THIS code block be `v-else-if="!getNameRequestNumber"` ?

Another way to resolve this is to test this component with a named amalgamation, a numbered amalgamation, a named non-amalgamation (eg, BC IA) and a numbered non-amalgamtion.

Reply:
I'm expecting you (James) to have tested the following 4 scenarios (and compared with Dev where appropriate):
1. named non-amalgamation (eg, IA or registration)
2. numbered non-amalgamation
3. named amalgamation
4. numbered amalgamation

I'm resolving this conversation. Please unresolve it (or start a new conversation) if you find anything amiss.